spec/mailers/user_mailer_spec.rb

```diff
@@ -608,6 +608,28 @@
         expect(email).to have_text_part_content(report.url)
       end
     end
+
+    describe "translation" do
+      before { Timecop.freeze("2022-03-14 13:27:09 +0000") }
+
+      after { Timecop.return }
+
+      it "formats the date rightfully in English" do
+        expect(email).to have_html_part_content("Sent at 01:27PM UTC Mon 14 March 2022.")
+        expect(email).to have_text_part_content("Sent at 01:27PM UTC Mon 14 March 2022.")
+      end
+
+      context "french" do
+        before { I18n.locale = "fr" }
+
+        after { I18n.locale = I18n.default_locale }
```

Comment:
I think you can skip this. We reset it here:

https://github.com/otwcode/otwarchive/blob/1f8558452e6984a41ace47074324013d2a3454f8/spec/spec_helper.rb#L67-L68